core/src/com/unciv/logic/city/CityExpansionManager.kt

```diff
@@ -10,11 +10,12 @@ class CityExpansionManager {
     @Transient
     lateinit var cityInfo: CityInfo
     var cultureStored: Int = 0
-
+    var tilesNotImproved: Int = 0
```

Comment:
Since we only use this from one place, and it's not something that's very computationally challenging, I would make this a getNumberOfUnimprovedTiles() function instead of a field